src/D2L.CodeStyle.TestAnalyzers/IgnoreAttribute/IgnoreAttributeAnalyzer.cs

```diff
@@ -41,6 +42,15 @@ internal sealed class IgnoreAttributeAnalyzer : DiagnosticAnalyzer {
 
 			foreach( var attribute in root.Attributes ) {
 				if( attribute.Name.ToString().Equals( "Ignore" ) ) {
+					var parentClassDeclaration = root.AncestorsAndSelf().OfType<ClassDeclarationSyntax>().ToImmutableArray().Single();

```

Comment:
Sorry, didn't get that, what do you mean *check what the ignore attribute is on*?

The ignore sometimes is not accompanied by a Test, i.e:
```
       [TestFixture]  
	[Ignore]   
	internal sealed class IgnoreSample {}
```